products/bpmsuite/_common/product.yml

```diff
@@ -5,9 +5,9 @@ has_installer: true
 current_version: 6.1.0.GA
 documentation_minor_version: 6.1
 xpaas_url: https://www.openshift.com/developers/jboss-bpms
-intro_video: https://vimeo.com/95505857
-vimeo_album: http://vimeo.com/album/2982446
-youtube_album: https://www.youtube.com/playlist?list=PLf3vm0UK6HKphPQWLX8SPJlPyqDhpZS4U
+intro_video: https://vimeo.com/142667410
```

Comment:
This is hosted on redhat's youtube, do you support that? When I tried it did not work so quickest fix is to push the original content to vimeo on my account.

BU product owners (PM, PMM and myself) want very much to update the content here that is very, very outdated.

Can we push this for now and I will push changes once hosted wherever you want it?